pkg/policymutation/policymutation.go

```diff
@@ -224,8 +224,8 @@ func GeneratePodControllerRule(policy kyverno.ClusterPolicy, log logr.Logger) (p
 
```

Comment:
missing "// scenario C"?

Reply:
C is handled in https://github.com/kyverno/kyverno/pull/1847/files/2704e40165e020a3b846cf477108220d8ce24e22#diff-15f5667117ad6d93290286aaa292507f75cc3aa9441f9b8f03bdfe1cd31cd6acR320.

Reply:
I meant in the comments....

Reply:
The comment was added to another place, not so obvious to see:
https://github.com/kyverno/kyverno/blob/2704e40165e020a3b846cf477108220d8ce24e22/pkg/policymutation/policymutation.go#L320-L321